react-native-sdk/package.json

```diff
@@ -63,6 +63,7 @@
         "@react-native-community/netinfo": "0.0.0",
         "@react-native-community/slider": "0.0.0",
         "@react-native-google-signin/google-signin": "0.0.0",
+        "abab": "0.0.0",
```

Comment:
Isn't this a normal dependency rather than a peer one?

Reply:
All good. Moved it to normal deps.